assets/js/app.js

```diff
@@ -22,6 +40,16 @@ $( document ).ready(function() {
                 cb(matches);
           };
     };
+    
+    var getMeta = $.ajax({
+                            async:false,
+                            url: 'https://gist.githubusercontent.com/xolf/2fc9efcd1b979d979ae4/raw/1be89cddb2304c1705dfab7ce37bd2963661db10/names.json',
+                            type:'get',
+                            data:{'GetConfig':'YES'},
+                            dataType:"JSON"
+                        }).responseJSON;
+    
+    console.log(getMeta);
```

Comment:
Debugging in master branch is generally not good idea. Also make the ajax calls `async`, there is no point of blocking the script.